build.sh

```diff
@@ -18,7 +18,7 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libraft pylibraft raft-dask docs tests bench clean --uninstall  -v -g -n --compile-libs --compile-nn --compile-dist --allgpuarch --no-nvtx --show_depr_warn -h --minimal-deps"
+VALIDARGS="clean libraft pylibraft raft-dask docs tests bench clean --uninstall  -v -g -n --compile-lib --allgpuarch --no-nvtx --show_depr_warn -h"
```

Comment:
So what does the `libraft` arg here do that is different from `--compile-lib`?

Reply:
Libraft installs the headers, `compile-lib` compiles the library. This keeps the build.sh consistent with other rapids projects. The cpp target is always libraft while the compiled shared lib is an option.